tool/grind.dart

```diff
@@ -146,6 +150,56 @@ Future<List<Map>> _buildSdkDocs(String sdkDocsPath, Future<String> futureCwd,
       workingDirectory: cwd);
 }
 
+Future<List<Map>> _buildTestPackageDocs(
+    String outputDir, Future<String> futureCwd,
+    [String label]) async {
+  if (label == null) label = '';
+  if (label != '') label = '-$label';
+  var launcher = new SubprocessLauncher('build-test-package-docs$label');
+  await launcher.runStreamed(sdkBin('pub'), ['get'],
```

Comment:
I think there's a grinder shorthand for `sdkBin('pub')` (but don't recall what it is).

Reply:
There is indeed (a class named [Pub](https://www.dartdocs.org/documentation/grinder/0.8.1/grinder.sdk/Pub-class.html)).  However grind's shorthands don't allow me to have the control I want for how they are executed. I've found it frustrating to debug a series of fork-execs that is partially hidden by "shorthands" that look like they aren't fork/execs.  This way, I can see exactly what it is doing both in the output of grinder and in the source code.